src/main/java/cz/codecamp/logger/formatters/Formatter.java

```diff
@@ -0,0 +1,46 @@
+package cz.codecamp.logger.formatters;
+
+import java.util.NoSuchElementException;
+
+public abstract class Formatter {
+
+    private String getTopDomainPackageName(String fullClassName) {
```

Comment:
Lepší než tyhle common metody řešit společným předkem je udělat si nějakou `FormatterUtil` classu a do ní dát jako statické metody.